### ISSUE TITLE
New Release Breaks Android Build

### ISSUE THREAD

**Issue by dmhood** [gre/react-native-view-shot#75]
W/ the new release I get the following Gradle/Java Errors:
```
.../node_modules/react-native-view-shot/android/src/main/java/fr/greweb/reactnativeviewshot/ViewShot.java:40: error: cannot find symbol
    private  ReactApplicationContext reactContext;
             ^
  symbol:   class ReactApplicationContext
  location: class ViewShot


.../node_modules/react-native-view-shot/android/src/main/java/fr/greweb/reactnativeviewshot/ViewShot.java:52: error: cannot find symbol
            ReactApplicationContext reactContext,
            ^
  symbol:   class ReactApplicationContext
  location: class ViewShot


.../node_modules/react-native-view-shot/android/src/main/java/fr/greweb/reactnativeviewshot/ViewShot.java:80: error: cannot find symbol
                reactContext.sendBroadcast(new Intent(Intent.ACTION_MEDIA_SCANNER_SCAN_FILE, Uri.parse(uri)));
                                               ^
  symbol:   class Intent
  location: class ViewShot


.../node_modules/react-native-view-shot/android/src/main/java/fr/greweb/reactnativeviewshot/ViewShot.java:80: error: cannot find symbol
                reactContext.sendBroadcast(new Intent(Intent.ACTION_MEDIA_SCANNER_SCAN_FILE, Uri.parse(uri)));
```

Looks to be what was added with https://github.com/gre/react-native-view-shot/commit/dd394a1356de95f6197119be3b52f7ec7852be95#diff-ed623a4868c175070504cf68f990095c

I re-checked the Android setup stuff and it looks good to go, unless I'm missing a new step/some dependency I don't know about?

**Comment by priezz** [comment]
+1

**Comment by kelset** [comment]
Same issue here. I had to roll back to 1.9.1 to fix.

**Comment by gre** [comment]
Oops sorry for that. i've released a `v1.10.1`
